File: code-examples/python/contents/update.py
import http.client

conn = http.client.HTTPConnection("api.marketcloud.it")

payload = "{\"title\" : \"My new blogpost\", \"text\" : \"This is the updated content of my new blogpost\"}"

headers = {
    'authorization': "YourPublicKey:YourToken",
    'accept': "application/json",
    'content-type': "application/json"
    }

conn.request("PUT", "/v0/contents/:id", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: code-examples/csharp/contents/update.cs
var client = new RestClient("http://api.marketcloud.it/v0/contents/:id");
var request = new RestRequest(Method.PUT);
request.AddHeader("content-type", "application/json");
request.AddHeader("accept", "application/json");
request.AddHeader("authorization", "YourPublicKey:YourToken");
request.AddParameter("application/json", "{\"title\" : \"My new blogpost\", \"text\" : \"This is the updated content of my new blogpost\"}", ParameterType.RequestBody);
IRestResponse response = client.Execute(request);